dashboard/app/controllers/pd/workshop_dashboard_controller.rb

```diff
@@ -4,13 +4,15 @@ class WorkshopDashboardController < ApplicationController
     def index
       @permission = nil
 
+      contact_rp = RegionalPartner.find_by(contact: current_user)
```

Comment:
OIC. However, these permissions dictate which options are available in the UI and we want to show these PLP options to all Regional Partner program managers. It should use the HABTM association for both. 

AFAIK we don't want to treat the contact differently than other program managers other than being the primary contact (email) associated with that partner. Outside the scope of this PR it might even be better to rename it `primary`, and/or have it point to the program_manager HABTM row instead of directly to a user.